.github/workflows/ci-sharktank.yml

```diff
@@ -85,7 +85,11 @@ jobs:
       - name: Run sharktank tests
         if: ${{ !cancelled() }}
         run: |
-          pytest -n 4 sharktank/ --durations=10
+          pytest \
+            -n 4 \
+            --durations=10 \
+            -m "not expensive" \
+            sharktank/
```

Comment:
Be careful when adding new tests, particularly if they are expensive. Developers should be able to run `pytest` and expect a reasonable default set of tests to run. Developers will not remember opt-out marks like this.

The filtering also didn't work here? https://github.com/nod-ai/shark-ai/actions/runs/13079961026/job/36501056330?pr=870 is still running on standard github-hosted runners after 2h+.

Reply:
The test failure is due to this required PR https://github.com/iree-org/iree-turbine/pull/418.

Reply:
I changed the default marker selection in `pyproject.toml` to exclude `expensive`.